core/src/test/java/edu/wpi/grip/core/serialization/ProjectTest.java

```diff
@@ -23,6 +24,7 @@
 import static junit.framework.TestCase.assertEquals;
 import static org.bytedeco.javacpp.opencv_core.*;
 
+@SuppressWarnings("PMD")
```

Comment:
# Not how this should be fixed. Add the specific suppressions to the specific methods.